controllers/apply-next/docs/validation-rules.md

```diff
@@ -16,11 +16,11 @@ This file documents validation rules, conditions, and error message text for all
 
 **Type**: date
 
-| Rule                              | Message                                        |
-| --------------------------------- | ---------------------------------------------- |
-| Required field                    | Enter a date                                   |
-| Must be a valid date              | Enter a real date                              |
-| At least 12 weeks into the future | Date must be at least 12 weeks into the future |
+| Rule                              | Message                                                                    |
+| --------------------------------- | -------------------------------------------------------------------------- |
+| Required field                    | Enter a date                                                               |
+| Must be a valid date              | Enter a real date                                                          |
+| At least 12 weeks into the future | Date you start the project must be after 25 July 2019/12 WEEKS INTO FUTURE |
```

Comment:
Is the "12 WEEKS INTO FUTURE" part meant to be included / in all-caps? Or is this just indicating that the date is dynamic?

Reply:
Glad to hear it. The all caps is just to indicate that the date is dynamic